### DISCUSSION TITLE
Welcome to ytdl-core Discussions!

### DISCUSSION THREAD

**Discussion by fent** [fent/node-ytdl-core#828] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas for the project, including new features (that fall into scope), optimizations, documentation, and direction for the project
  * Share any projects you've created with ytdl-core
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about how you use ytdl-core

**Comment by bward-dev1** [comment]
Does this work against SABR?

**Comment by joyboi-ai** [comment]
Yes it does
